docs/csharp/language-reference/operators/nameof.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "nameof operator - C# reference"
+ms.custom: seodec18
+ms.date: 07/12/2019
+f1_keywords:
+  - "nameof_CSharpKeyword"
+  - "nameof"
+ms.assetid: 33601bf3-cc2c-4496-846d-f9679bccf2a7
+---
+# nameof operator (C# reference)
+
+The `nameof` operator obtains the name of a variable, type, or member as the string constant:
+
+[!code-csharp-interactive[nameof operator](~/samples/csharp/language-reference/operators/NameOfOperator.cs#Examples)]
+
+As the preceding example shows, in the case of a type and a namespace, the produced name is not [fully qualified](~/_csharplang/spec/basic-concepts.md#fully-qualified-names).
```

Comment:
@BillWagner I'm not technically correct here. The sentence should finish with "...unless a type or a namespace is the member of the global namespace." However I didn't want to over-complicate things. Is the trimmed version fine?

Reply:
I like the way you've worded this. I was looking through the spec this morning to find good language as well. I think a trimmed version is fine, but I'll suggest a bit of weasel words:

```suggestion
As the preceding example shows, in the case of a type and a namespace, the produced name is usually not [fully qualified](~/_csharplang/spec/basic-concepts.md#fully-qualified-names).
```

Reply:
@BillWagner perfect! I've looked for that, but overlooked:) Thanks!